lib/send_text.rb

```diff
@@ -0,0 +1,34 @@
+require 'twilio-ruby'
+
+class SendText
+
+  attr_reader :customers
+
+  def confirmation_text
+    sender
+  end
+
+  private
+
+    def sender
+
+      accountSID = 'ACc7752d251c147e6953e3236fb686dc5b'
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)